source/common/network/utility.h

```diff
@@ -67,7 +67,15 @@ class Utility {
 
   /**
    * Parse an internet host address (IPv4 or IPv6) AND port, and create an Instance from it. Throws
```

Comment:
I wonder outside this PR whether we should just made this part of the `Address` interface.